README.md

```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
 
 The following available sensor types should be used.
 
-- Occupancy
-- Motion
-- PIR
-- Light
+-- Environmental Sensor Types
 - Temperature
-- Humidity
-- CO2
-- VOC
+- Precipitation
+- Soil moisture
+- Soil temperature
 - Barometric Pressure
+- Vapor Pressure
 - Noise
-- Battery Percent
-- Battery Voltage
+- Light
+- Solar radiation
+- Lightning strike detector
+- Wind speed
+- Wind direction
 
-## Available data keys
 
-The following available data keys should be used.
+-- Air quality Sensor Types
+- Humidity
+- CO2
+- eCO2
+- CO
+- VOC
+- PM1.0
+- PM2.5
+- PM4.0
+- PM10
+
+-- Electricity Sensor Types
+- Energy
+- LEM
+- External Analog Input
+- do_1_lora
+- do_2_lora
+- do_1_error
+- do_2_error
+- do_1
+- do_2
+- manually
+- Light State
 
+-- Magnetic & infrared Sensor Types
+- Reed
 - Occupancy
 - Motion
 - PIR
-- Light
+
+-- Acceleration Sensor Types
+- Acceleration
+- Gyro
+- Magnetometer
+
+-- GPS Sensor Types
+- Latitude
+- Longitude
+- Altitude
+
+-- People Counting Sensor Types
+- Forward
+- Backwards
+- Gender
+- FaceMask
+- Dwelltime
+- Average Waiting Time
+- Queue Depth
+
+-- Diverse Sensor Types
+- Button
+- Distance
+- Vibration
+- Water leak
+- Anomaly
+- Status
+- Image
+
+
+## Available data keys
+
+The following available data keys should be used.
```

Comment:
does it really make sense to include all data keys here, I would opt for just declaring the most common ones

Reply:
Cutted some keys. If there are more changes required maybe it would be good to have a call.